apps/researcher/src/app/[locale]/(objects)/objects/[id]/object-lists-menu.tsx

```diff
@@ -123,14 +124,12 @@ export default function ObjectListsMenu({objectId}: ObjectListsMenuProps) {
   const t = useTranslations('ObjectDetails');
   const {user} = useUser();
 
-  if (!user || !user.organizationMemberships.length) {
+  const communities = useUserCommunities();
```

Comment:
For consistency, rename `communities` to `userCommunities` or rename `useUserCommunities` to `useCommunities`?